client/src/components/App.jsx

```diff
@@ -9,58 +9,43 @@ import fetcher from '../fetchers';
 
 export default function App() {
   // PRE-FETCH EMPTY INITIAL VALUE //
-  const initProd = {
-    id: 0,
-    campus: '',
-    name: '',
-    slogan: '',
-    description: '',
-    category: '',
-    default_price: '',
-    created_at: '',
-    updated_at: '',
-    features: [],
-  };
+  const initFeature = { id: 40350, name: null };
 
-  // // STATE DATA //
-  // const [featuredProduct, setFeaturedProduct] = useState(initProd);
-
-  // // INITIALIZATION //
-  // useEffect(() => {
-  //   fetcher.getProductById(40350)
-  //     .then((result) => setFeaturedProduct(result.data))
-  //     .catch((err) => console.error('initial fetch: ', err));
-  // }, []);
-
-  const [featuredProduct, setFeaturedProduct] = useState(initProd);
+  // STATE
+  const [feature, setFeature] = useState(initFeature);
   const [styles, setStyles] = useState(null);
   const [reviews, setReviews] = useState(null);
   const [reviewMeta, setReviewMeta] = useState(null);
+  const [relatedIdList, setRelatedIdList] = useState(null);
 
   // INITIALIZATION //
   useEffect(() => {
     axios.all([
-      fetcher.getProductById(40350),
-      fetcher.overview.getStylesById(40350),
-      fetcher.ratings.getReviews(40350),
-      fetcher.ratings.getReviewMeta(40350),
+      fetcher.getProductById(feature.id),
+      fetcher.getProductStyle(feature.id),
+      fetcher.getReviews(feature.id),
+      fetcher.getReviewMeta(feature.id),
+      fetcher.getRelatedProduct(feature.id),
     ])
       .then(axios.spread((...data) => {
-        setFeaturedProduct(data[0].data);
+        setFeature(data[0].data);
         setStyles(data[1].data);
         setReviews(data[2].data.results);
         setReviewMeta(data[3].data);
+        setRelatedIdList(data[4].data);
       }))
       .catch((err) => console.error(err));
-  }, []);
+  }, [feature.id]);
+
+  if (!feature.name) return <div />;
 
   return (
     <div>
       <Overview product={featuredProduct} styles={styles} reviews={reviews} />
```

Comment:
```suggestion
      <Overview product={feature} styles={styles} reviews={reviews} />
```
See line 15, state's name is `feature`.

Reply:
I do understand it's confusing. My idea for the name came from before understanding there were properties keyed as 'feature'. I am not opposed to renaming this state, but I do not want this component to break on merge, hence my suggestion.